### DISCUSSION TITLE
What does output event "Void" with Value 0 means?

### DISCUSSION THREAD

**Discussion by arielhasidim** [MIT-LCP/mimic-code#1380] [MIMIC-General] [answered]
What does output event "Void" with Value 0 means?

When running in bigquery:
```
select * from `physionet-data.mimiciv_icu.outputevents`
where itemid = 226560 -- "Void"
  and value = 0`
```
I get 3,293 results.

(Or maybe this should be an issue?)

**Comment by marymlucas** [comment, marked as answer]
Hi @arielhasidim I'm not sure if this is the correct answer, but if these aren't erroneous entries then my guess would be that it refers to a urine output of 0ml. In medicine the word "void" is often used interchangeably with the word "urinate."

**Reply by arielhasidim** [reply]
Thank you Lucas.
This is exactly the problem, I'm treating void as urination and this why output event of 0ml urination doesn't make sense. When charting urination you chart positive events, not non-events. 🤷🏾‍♂️

**Reply by marymlucas** [reply]
Hi @arielhasidim, 
A urine output of 0ml is clinically an event, depending on context.  From my past clinical experience, if you're monitoring a patient closely and need to document urine output at a regular frequency, e.g hourly, or every 6 hrs, etc., you would chart 0ml if the patient did not void during that time interval.  It's still considered an event even if the output is 0ml as that could indicate a medical issue eg renal failure.  I'm not saying this is the case with these entries, but I would hesitate to consider them non-events.

Hope this helps.

Mary

**Reply by arielhasidim** [reply]
Thank you very much Mary, this seems like a reasonable explanation.

@alistairewj Can you also please approve this as truthy values and not charting mistakes, and that it would make sense to interpret this as Mary explained?
